sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -840,3 +840,46 @@ case class GreaterThanOrEqual(left: Expression, right: Expression)
 
   protected override def nullSafeEval(input1: Any, input2: Any): Any = ordering.gteq(input1, input2)
 }
+
+/**
+ * Test the value of an expression is true, false, or unknown.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(expr, booleanValue) - Returns true if `expr` equals booleanValue, " +
+    "or false otherwise.",
+  arguments = """
+    Arguments:
+      * expr - a boolean expression
+      * booleanValue - a boolean value represented by a string. booleanValue must be one
+          of TRUE, FALSE and UNKNOWN.
+  """,
+  examples = """
+    Examples:
+    > SELECT _FUNC_(1 > 2, true);
+       false
+    > SELECT _FUNC_(2 > 1, true);
```

Comment:
Does PostgreSQL has a function like this, @beliefer ?
This guess your example will not work since we didn't register this as a function yet.
Until now, this PR only support `IS TRUE|FALSE|UNKNOWN` SQL syntax.

Reply:
@dongjoon-hyun I checked the latest docs of PostgreSQL and can't found function about this.
ref : https://www.postgresql.org/docs/11/functions-comparison.html
Maybe we not need to register this as a function, but register function named as `IsTrue` and `IsFalse` is OK too.

Reply:
Yes. Then, let's remove this `@ExpressionDescription` annotation. We don't need this.

Reply:
I think its ok to target `IS TRUE|FALSE|UNKNOWN` only in this pr as Dongjoon suggested. `IsTrue/IsFalse` is out-of-scope here.

Reply:
@dongjoon-hyun @maropu Thanks for your suggestion. I will remove it.